shared/src/common/types/http/http-options.type.ts

```diff
@@ -1,11 +1,12 @@
-import { ContentType, HttpMethod } from 'common/enums/enums';
+import { ContentType, HttpMethod } from '~/common/enums/enums';
 
 type HttpOptions = {
   method: HttpMethod;
   contentType: ContentType;
   payload: BodyInit | null;
   hasAuth?: boolean;
   queryString?: Record<string, unknown>;
+  headers?: Record<string, string>;
```

Comment:
@what1s1ove `Headers` is not defined for Node apps, since it's the part of `fetch API`
That's why I decided to use `Record`, so it's appropriate both for FE and BE
Should I leave it as `Record`?

Reply:
resolved